test/integration/response-test.js

```diff
@@ -8,6 +8,40 @@ const expect = require('chai').expect;
 describe('Response', () => {
   afterEach(() => mockyeah.reset());
 
+  it('should not allow for more than one response type', () => {
+    const payloadKeys = ['fixture', 'filePath', 'html', 'json', 'text'];
+    const optionSets = [
+      {
+        fixture: './test/fixtures/some-data.json',
+        filePath: './test/fixtures/some-data.csv'
+      },
+      {
+        fixture: './test/fixtures/some-data.json',
+        filePath: './test/fixtures/some-data.csv',
+        html: '<body></body>'
+      },
+      {
+        fixture: './test/fixtures/some-data.json',
+        filePath: './test/fixtures/some-data.csv',
+        html: '<body></body>',
+        json: '{"test": "json"}'
+      },
+      {
+        fixture: './test/fixtures/some-data.json',
+        filePath: './test/fixtures/some-data.csv',
+        html: '<body></body>',
+        json: '{"test": "json"}',
+        text: 'test text'
+      }
+    ];
+
+    optionSets.forEach(optionSet => {
+      expect(() => {
+        mockyeah.get('/service/end/point', optionSet);
+      }).to.throw('Response options must not include more than one of the following: ' + payloadKeys.join(', '));
+    });
+  });
+
```

Comment:
@ryanricard, doing the permutations for these (e.g. 5 options, pick 2, 5 options, pick 3, 5 options, pick 4, 5) in a recursive manner proved a bit challenging and maybe a bit overdone for what we are trying to accomplish here. 

Instead, I propose that we validate that 2, 3, 4, and 5 options are not allowed, instead of testing each permutation. Would this be a fair compromise?